tools/docker/docker-installation.md

```diff
@@ -8,11 +8,35 @@ order: 1
 
 ## Installation
 
-On Fedora 22 and above install the `docker` package:
+Install the `docker-ce` package using the Docker repository:
 
+To install the dnf-plugins-core package (which provides the commands to manage your DNF repositories) and set up the stable repository.
+
+```
+$ sudo dnf -y install dnf-plugins-core
+```
+To add the `docker-ce` repository
+
+```
+$ sudo dnf config-manager \
+    --add-repo \
+    https://download.docker.com/linux/fedora/docker-ce.repo
+```
+
+To install the docker engine 
+
+```
+$ sudo dnf install docker-ce docker-ce-cli containerd.io
 ```
-$ sudo dnf install docker
+
+Afterwards you need to enable the backward compatability for Cgroups
+
 ```
+sudo grubby --update-kernel=ALL --args="systemd.unified_cgroup_hierarchy=0"
```

Comment:
Well, this is a system wide change? Could it break something else on the system?

If yes, I'd put a big warning before it.

Reply:
Since it is referring to backward compatibility and since it is in the official Docker Documentation, I would assume no. However, I have not dived deep enough into the system to know for sure.

Reply:
I assume you've tried it, did it break anything on the system for you?

Reply:
Not that I have found thus far. I have not seen any ill effects from this line.